packages/gatsby-cli/src/reporter/index.js

```diff
@@ -399,9 +399,9 @@ const reporter: Reporter = {
   _setStage() {},
 }
 
-console.log = (...args) => reporter.log(util.format(...args))
-console.warn = (...args) => reporter.warn(util.format(...args))
-console.info = (...args) => reporter.info(util.format(...args))
-console.error = (...args) => reporter.error(util.format(...args))
+// console.log = (...args) => reporter.log(util.format(...args))
```

Comment:
These make debugging reducers _so painful_ 😒(you can't dispatch in reducers and structured logging makes console.log dispatch)